static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js

```diff
@@ -15,11 +15,19 @@ export const acquisitionsBannerFivTemplate = (
                 ${closeCentralIcon.markup}
             </button>
         </div>
-        <div class="fiv-banner__headline">${
+        ${
             params.titles
-                ? `${params.titles[0]} THEN ${params.titles[1]}`
-                : 'NOPE'
-        }</div>
+                ? `<div class="fiv-banner__headline">
+            <div class="fiv-banner__headline1">
+    ${params.titles[0]}
+            </div>
+            <div class="fiv-banner__headline2">
+                ${params.titles[1]}
```

Comment:
I could have done this more like a for each rather than assuming exactly two sections, but for the campaign it's hard coded

Reply:
Sounds like a good call, esp since there’s no more lines of styling either - that said the indents look wonky in this diff!

Reply:
make fix is a law unto itself in my machine!  I think that's the way to keep it happy...